app/assets/scripts/components/new/select.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+import { _cs } from '@togglecorp/fujs';
```

Comment:
@frozenhelium we are already using a library called `classnames` in the project that seems to do something very similar. You can see an example of usage here: https://github.com/IFRCGo/go-frontend/blob/develop/app/assets/scripts/components/fold.js#L22

It's used in many components and it would be nice to stay consistent. I'd rather stick with the existing library for now -- if there's a good reason to use the new library, let's make a ticket to move over all the existing usage of the `classnames` library to use the `_cs` method from `@togglecorp/fujs`. It would be really nice to not have multiple ways of doing this across the code-base :-)

Reply:
Sure, I'll check if they have same set of features and take action accordingly